src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  constructor(props) {
```

Comment:
- [OPTIONAL] It is always a good practice to make sure you fix all linter errors instead of disabling them, Linter is a tool for identifying and reporting errors when you use standard rules. 💯 😉 

- [OPTIONAL] You have also done the same thing in the `App` component. Kindly fix them.
- **Hint**: I guess you are trying to make a way through the class-based component. I will advise you to go with React Pure component. [ReadMore](https://www.geeksforgeeks.org/reactjs-pure-components/)

eg: 
```.js
export default class Calculate extends React.PureComponent{
   render(){
      return <h1>blablabla</h1>;
   }
}
```

In that example, App is still a class-based component